doc/api.yml

```diff
@@ -0,0 +1,844 @@
+          $ref: '#/components/responses/500'
+  '/archiving/trash':
+    post:
+      tags:
+        - Posting
+      summary: Delete post.
+      operationId: archivePost
+      requestBody:
+        description: Post to update
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Post'
+      responses:
+        '200':
+          description: Post deleted by identifier.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Post'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '429':
+          $ref: '#/components/responses/429'
+        '500':
+          $ref: '#/components/responses/500'
+  '/posts/{id}/reaction':
+    post:
+      tags:
+        - Posting
+      summary: React to a post by identifier.
+      operationId: reactToPostById
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: reaction_type
+          in: query
+          required: true
+          schema:
+            $ref: '#/components/schemas/ReactionType'
+      responses:
+        '201':
+          description: Post reacted.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Reaction'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '429':
+          $ref: '#/components/responses/429'
+        '500':
+          $ref: '#/components/responses/500'
+
+  '/post/{id}/comments/{comment_id}/reaction':
+    post:
+      tags:
+        - Posting
+      summary: React to a comment by identifier.
+      operationId: reactToComment
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: comment_id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: reaction_type
+          in: query
+          required: true
+          schema:
+            $ref: '#/components/schemas/ReactionType'
+      responses:
+        '201':
+          description: Post reacted.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Reaction'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '429':
+          $ref: '#/components/responses/429'
+        '500':
+          $ref: '#/components/responses/500'
+
+        ## Commenting resources
+  '/posts/{id}/comments':
+    get:
```

Comment:
L1 - You forgot to add the request that add or modify a comment:
PUT - /posts/{id}/comments

check the reference picture in the Jira ticket to make the necessary modifications